jobs/cancel_orders.ts

```diff
@@ -45,22 +47,22 @@ const cancelOrders = async bot => {
     for (const order of activeOrders) {
       const buyerUser = await User.findOne({ _id: order.buyer_id });
       const sellerUser = await User.findOne({ _id: order.seller_id });
-      const i18nCtxBuyer = await getUserI18nContext(buyerUser);
-      const i18nCtxSeller = await getUserI18nContext(sellerUser);
+      const i18nCtxBuyer = await getUserI18nContext(buyerUser!);
+      const i18nCtxSeller = await getUserI18nContext(sellerUser!);
```

Comment:
@Mersho can you explain this please? /cc @aarani

Reply:
> @Mersho can you explain this please? /cc @aarani

I'll fix it. The null-check I did previously with this operator was completely incorrect.

Reply:
I should throw, but I am changing this like the original repo

`if (buyerUser === null || sellerUser === null) return;`

Reply:
wouldnt that change behaviour? /cc @aarani